cancer_ai/validator/competition_manager.py

```diff
@@ -9,10 +9,15 @@
 from .dataset_manager import DatasetManager
 from .model_run_manager import ModelRunManager
 
+from  .competition_handlers.melanoma_handler import MelanomaCompetitionHandler
+
 
 COMPETITION_MAPPING = {
```

Comment:
to wywal

Reply:
Sure, but how to go about COMPETITION_HANDLER_MAPPING?

Reply:
it's fine, just delete COMPETITION_MAPPING, it's not used

Reply:
I decided to use it in evaluate function in competition_menager. I am sorry for the confiusion because COMPETITION_MAPPING is different from COMPETITION_HANDLER_MAPPING. I could pick better names

Reply:
Fixed 👍